packages/config/src/layer2s/dydx.ts

```diff
@@ -107,7 +109,7 @@ export const dydx: Layer2 = {
         name: 'GpsStatementVerifier',
```

Comment:
use `getMainContractDetails`, what about the other hardcoded contracts? can we get them from discovery?

Reply:
Also, please let me tackle the verifier in https://linear.app/l2beat/issue/L2B-1135/add-sharp-provers-to-starkex-configs